website/templates/emails/withdrawal_request_granted.html.mako

```diff
@@ -22,26 +22,41 @@
         % endif
     % else:
         % if is_requester:
-            Your request to withdraw your ${document_type} <a href="${reviewable.absolute_url}">"${reviewable.title}"</a> from ${reviewable.provider.name} has been approved by the service moderators.
-            <br>
-            The ${document_type} has been removed from ${reviewable.provider.name}, but its metadata is still available: title of the withdrawn ${document_type}, its contributor list, abstract, tags, DOI, and reason for withdrawal (if provided).
-            <br>
+            % if document_type == 'preprint':
+                Your request to withdraw your preprint <a href="${reviewable.absolute_url}">"${reviewable.title}"</a> from ${reviewable.provider.name} has been approved by the service moderators.
+                <br>
+                The registration has been removed from ${reviewable.provider.name}, but its metadata is still available: title of the withdrawn ${document_type}, its contributor list, abstract, tags, DOI, and reason for withdrawal (if provided).
```

Comment:
_registration_ has been removed? I think this is still in the preprint branch. (see above comment about doing bigger branches to consolidate preprint vs registration text)